api/neuronaApp/urls.py

```diff
@@ -11,4 +11,10 @@
     path("register/", views.RegisterView.as_view(), name="register"),
     path("login/", views.LoginView.as_view(), name="login"),
     path("", include(router.urls)),
+    path('spaces/', views.space_view.get_spaces, name='get_spaces'),
+    path('spaces/<int:pk>/', views.space_view.get_space, name='get_space'),
+    path('spaces/create', views.space_view.create_space, name='create_space'),
+    path('spaces/update/<int:pk>', views.space_view.update_space, name='update_space'),
+    path('spaces/delete/<int:pk>', views.space_view.delete_space, name='delete_space'),
+    path('spaces/post', views.space_view.create_space, name='create_space'),
```

Comment:
use the **same** route with different HTTP methods, calling different views to respect REST convention, and always add trailing slash `/` 

```suggestion
    path('spaces/', views.space_view.create_space, name='create_space'),
    path('spaces/<int:pk>/', views.space_view.update_space, name='update_space'),
    path('spaces/<int:pk>/', views.space_view.delete_space, name='delete_space'),
```